roles/keystone/tasks/main.yml

```diff
@@ -31,6 +31,15 @@
   notify:
     - restart keystone services
 
+- name: install ldap pip reqs in package venv
```

Comment:
So I looked at the giftwrap documentation, the real problem seems to be that requirements.txt does not list python-ldap and ldappool in requirements.txt(https://github.com/openstack/keystone/blob/stable/juno/requirements.txt) . 
Following is a bug that was opened for ubuntu 
https://bugs.launchpad.net/ubuntu/+source/keystone/+bug/1364854 

Ive opened a separate bug for keystone
https://bugs.launchpad.net/keystone/+bug/1487671